sdk/push-fcm/src/main/java/com/klaviyo/pushFcm/KlaviyoNotification.kt

```diff
@@ -58,7 +58,6 @@ class KlaviyoNotification(private val message: RemoteMessage) {
         internal const val BODY_KEY = "body"
         internal const val URL_KEY = "url"
         internal const val IMAGE_KEY = "image_url"
-        internal const val CLICK_ACTION_KEY = "click_action"
```

Comment:
I added this key in the very first version in the event we supported other types of intent actions. I don't think its something we'd ever need to use after all.